j2k/testData/fileOrElement/methodCallExpression/stringMethods.kt

```diff
@@ -7,7 +7,7 @@ internal class A {
     @Throws(Exception::class)
     fun constructors() {
         String()
-        // TODO: new String("original");
+        "original"
```

Comment:
@yole yes, you're right. Saying that I'm not sure about a provided solution without providing any helpful clues or suggestions was inappropriate; I won't.